ui/src/dashboards/actions/thunks.ts

```diff
@@ -126,6 +120,7 @@ export const cloneDashboard = (dashboard: Dashboard) => async (
     )
 
     const dash: Dashboard = entities[result]
```

Comment:
is this an error right here?

Reply:
great catch.  will create a util function to do this for us.